pkg/catalogv2/git/download.go

```diff
@@ -82,9 +82,13 @@ func Update(secret *corev1.Secret, namespace, name, gitURL, branch string, insec
 		return commit, fmt.Errorf("update failure: %w", err)
 	}
 
-	if changed, err := git.remoteSHAChanged(branch, commit); err != nil || !changed {
+	changed, err := git.remoteSHAChanged(branch, commit)
```

Comment:
@nicholasSUSE Can we stick to the previous implementation and make another issue and then PR for this mistake ?

Thanks 😄

Reply:
I am not really sure what you are talking about.

Do you mean the change from Ensure(commit) -> Ensure(branch)?
This is still the Ensure(commit), I am not changing this behavior at this PR.